com.reprezen.swagedit/src/com/reprezen/swagedit/messages.properties

```diff
@@ -21,9 +21,9 @@ error_additional_properties_not_allowed = object has properties %s which are not
 error_required_properties = object has missing required properties %s
 error_duplicate_keys = Duplicate key: the object has more than one property named %s
 error_cannot_read_content = Unable to read content. It may be invalid YAML
-error_invalid_reference = Invalid Reference - SwagEdit was unable to resolve the reference. \n\
+error_missing_reference=Invalid Reference - SwagEdit was unable to resolve the reference. \n\
 The value must be a valid JSON Reference (for external references) or JSON Pointer (for local references), and must resolve to an object of the expected type.
-
+error_invalid_reference=Invalid Reference Syntax - The referenced path or URI may contain invalid characters. 
```

Comment:
In https://github.com/RepreZen/SwagEdit/pull/151#discussion_r70855192 @tedepstein wrote: 
> It would be better to change the first line to this: <...>
The second line can remain the same.

I interpret it as that we need a two-line message:
> Invalid Reference Syntax - The referenced path or URI may contain invalid characters.  
The value must be a valid JSON Reference (for external references) or JSON Pointer (for local references), and must resolve to an object of the expected type.

@tedepstein , can you please confirm that this is the message you suggested?

While the new code has only the first line and omits the second line:
<img width="590" alt="screen shot 2016-07-15 at 9 49 09 am" src="https://cloud.githubusercontent.com/assets/644582/16876408/c21e2c04-4a71-11e6-9d95-64e01ca0c8d3.png">